libproject/test/libprojecttest.cpp

```diff
@@ -178,13 +179,13 @@ class TestAddRegularSubprojectsToNested : public TestSkeleton {
     json contentReferenceWithPairOfNewRegularSubprojects;
 
 
-    string pathToOneRegularSingleSubproject;
-    string pathToOneRegularNestedSubproject;
-    vector<string> pathToBrokenSubproject;
-    vector<string> pathsToPairOfRegularSubprojects;
-    vector<string> pathsToPairOfRegularAndBrokenSubprojects;
-    vector<string> pathsToPairOfBrokenSubprojects;
-    vector<string> pathsToPairOfRegularAndEmptySubprojects;
+    FileSetLoader::Path pathToOneRegularSingleSubproject;
+    FileSetLoader::Path pathToOneRegularNestedSubproject;
+    FileSetLoader::Path pathToBrokenSubproject;
+    FileSetLoader::Subprojects pathsToPairOfRegularSubprojects;
```

Comment:
> pathsToPairOfRegularSubprojects

now this name is redundant, so it can be replaced just to
```cpp
FileSetLoader::Subprojcts pairOfRegular; 
//or
FileSetLoader::Subprojects regular; //because pair is'n good quantor (it's two or above)
```